src/Arduino_NBConnectionHandler.cpp

```diff
@@ -113,7 +123,7 @@ void NBConnectionHandler::update() {
       case NetworkConnectionState::CONNECTING: {
           // NOTE: Blocking Call when 4th parameter == true
           NB_NetworkStatus_t networkStatus;
-          networkStatus = gprs.attachGPRS();
+          networkStatus = gprs.attachGPRS(apn, login, pass, true);
```

Comment:
I don't see a default value for `apn`, `login`, or `pass` do they default to NULL or empty string?

Reply:
not sure, but I think they're `NULL`

Reply:
I highly recommend they be explicitly initialized.

Reply:
How would you initialize those values? Can I do that in the signature?

Reply:
Also I don’t think this compiles, here is the signature: https://github.com/arduino-libraries/MKRNB/blob/master/src/GPRS.h#L58

The parameters should be passed to ‘nbAccess.begin(...)’.